src/auction/src/orders.mo

```diff
@@ -191,9 +191,9 @@ module {
 
       if (not isPartial) {
         assetInfo.totalExecutedOrders += 1;
-        order.userInfoRef.loyaltyPoints += C.LOAYLTY_REWARD.ORDER_FULFILLMENT_APPENDIX;
       };
-      order.userInfoRef.loyaltyPoints += quoteVolume * C.LOAYLTY_REWARD.ORDER_FULFILLMENT_COEFFICIENT;
+      let loayltyVolumeReward = Float.fromInt(quoteVolume) * C.LOAYLTY_REWARD.ORDER_VOLUME |> Int.abs(Float.toInt(_));
```

Comment:
Please fix typo "loaylty" to "loyalty" everywhere.

Reply:
Let's do this without Floats:
```
let loyaltyVolumeReward = quoteVolume / 100_000;
```

Reply:
Fixed